Lab1.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d94081c0-6bc7-4a47-801a-56016395d2e7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 1A.\n",
+    "# A normal distribution is where data trends around a central value, and the mean, median, and mode are all equal. \n",
+    "# A z-score is how many standard deviations away from the mean a data point is.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2211347d-d4a6-4719-b079-34beebdddcd4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#1B\n",
+    "import numpy as np\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "from scipy import stats\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = (15,10)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "696a3d73-8d97-47e5-aa17-2d3324a19907",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8413447460685429\n",
+      "0.9772498680518208\n",
+      "0.9986501019683699\n"
+     ]
+    }
+   ],
+   "source": [
+    "x = stats.norm.cdf(1)\n",
+    "print(x)\n",
+    "\n",
+    "y = stats.norm.cdf(2)\n",
+    "print(y)\n",
+    "\n",
+    "z = stats.norm.cdf(3)\n",
+    "print(z)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "5b14bc92-c59c-4f40-9ffa-a8877eb1f163",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.9985762706156592\n",
+      "1.9953933101678245\n",
+      "3.090232306167813\n"
+     ]
+    }
+   ],
+   "source": [
+    "#1C\n",
+    "x = stats.norm.ppf(.841)\n",
+    "print(x)\n",
+    "y = stats.norm.ppf(.977)\n",
+    "print(y)\n",
+    "z = stats.norm.ppf(.999)\n",
+    "print(z)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "1ca26205-f9db-4707-8e2d-5fa314b2f8d4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#1D\n",
+    "# You get a negative value of sigma when the probability is larger than 1/2."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "98c1c2c6-5f4c-4e7e-bbe1-f90192993bbf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#2A\n",
+    "# A log-normal distribution is a continuous probability distribution of a random variable whose logarithm is normally distributed.\n",
+    "# For a function Y = ln(X), if Y had would have a normal distribution."
```

Comment:
I don't understand the second sentence of 2A. Do you mean to say that if X is a random variable obeying a log-normal distribution then Y = ln(X) will obey a normal distribution?